doc/whatsnew/2/2.14/full.rst

```diff
@@ -14,6 +14,8 @@ Release date: TBA
 
   Closes #2270
 
+* Allow suppressing ``undefined-loop-variable`` without raising ``useless-suppression``.
```

Comment:
```suggestion
* Allow suppressing ``undefined-loop-variable`` and ``undefined-variable`` without raising ``useless-suppression``.
```
Let's only apply this once the github actions are green for the previous commit:)